### PR TITLE
Fix global search results panel on mobile

### LINKED ISSUE #7092: Global search results panel overflows screen on mobile

### PR DESCRIPTION
**Pull request by rasasak**
#7092
Global search containew full width on small displays
(before it hat SM breakpoint)

Before:

After:

- [x] Changes have been thoroughly tested to not break existing functionality.
- [nothing to document ] New functionality has been documented or existing documentation has been updated to reflect changes.
- [x] Visual changes are explained in the PR description using a screenshot/recording of before and after.

### CODE REVIEW

**Comment by zepfietje** [comment]
Thanks for this PR, @rasasak.
Could you add margin around the panel so it doesn't stick to the screen edges?

File: packages/panels/resources/views/components/global-search/results-container.blade.php [hunk 1]
```diff
@@ -23,7 +23,7 @@
     x-transition:leave-end="opacity-0"
     {{
         $attributes->class([
-            'fi-global-search-results-ctn absolute end-0 z-10 mt-2 max-h-96 w-screen max-w-sm overflow-auto rounded-lg bg-white shadow-lg ring-1 ring-gray-950/5 transition dark:bg-gray-900 dark:ring-white/10',
+            'fi-global-search-results-ctn absolute start-0 sm:start-auto sm:end-0 z-10 mt-2 max-h-96 w-full sm:w-screen sm:max-w-sm overflow-auto rounded-lg bg-white shadow-lg ring-1 ring-gray-950/5 transition dark:bg-gray-900 dark:ring-white/10',
```
Comment (zepfietje) [inline packages/panels/resources/views/components/global-search/results-container.blade.php]: Should probably use `inset-x-` (see my other comment).

**Review by zepfietje — CHANGES_REQUESTED** [review]

**Comment by rasasak** [comment]
@zepfietje  yeah you are right !  
how big inset ?  I set "4" .. same as widgets, etc...
but in my opinion  inset-x-3 have nice 3d efect .. it is  little bit bigger, because it is little closer to user 

or another size ?

inset-x-4:


inset-x-3:

**Comment by zepfietje** [comment]
Thanks, @rasasak!

**Pull request merged**